conans/client/conf/__init__.py

```diff
@@ -12,13 +12,13 @@
 
 default_settings_yml = """
 # Only for cross building, 'os_build/arch_build' is the system that runs Conan
-os_build: [Windows, WindowsStore, Linux, Macos, FreeBSD, SunOS]
-arch_build: [x86, x86_64, ppc32, ppc64le, ppc64, armv5el, armv5hf, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, s390, s390x]
+os_build: [Windows, WindowsStore, Linux, Macos, FreeBSD, SunOS, AIX]
+arch_build: [x86, x86_64, ppc32, ppc64le, ppc64, armv5el, armv5hf, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, s390, s390x, powerpc, rs6000]
 
 # Only for building cross compilation tools, 'os_target/arch_target' is the system for
 # which the tools generate code
-os_target: [Windows, Linux, Macos, Android, iOS, watchOS, tvOS, FreeBSD, SunOS, Arduino]
-arch_target: [x86, x86_64, ppc32, ppc64le, ppc64, armv5el, armv5hf, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, s390, s390x]
+os_target: [Windows, Linux, Macos, Android, iOS, watchOS, tvOS, FreeBSD, SunOS, AIX, Arduino]
+arch_target: [x86, x86_64, ppc32, ppc64le, ppc64, armv5el, armv5hf, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, s390, s390x, powerpc, rs6000]
```

Comment:
I think the ppc32, ppc64.. etc already meant PowerPC

Reply:
Basically yes but python and uname show different values for ppc on Linux and AIX:

Linux:
```
~$ uname -a
Linux xxx 2.6.32-754.el6.ppc64 #1 SMP Thu May 24 18:29:49 EDT 2018 ppc64 ppc64 ppc64 GNU/Linux
~$ uname -s
Linux
~$ uname -p
ppc64
~$ uname -m
ppc64
```

AIX:
```
~$ uname -a
AIX xxx 1 7 00FB91F44C00
~$ uname -s
AIX
~$ uname -p
powerpc
~$ uname -m
00FB91F44C00
```

Thats why I added rs6000/powerpc. Previously I already was using ppc32/ppc64. Then I had a look at the AutoTools and config.guess which generates the gnu triplets like:

Linux:
```
~$ ./config.guess 
powerpc64-unknown-linux-gnu
```

AIX:
```
~$ ./config.guess
powerpc-ibm-aix7.1.3.0
```

Reply:
We need to keep the same meaning for the settings, and maybe it was not the best name, but yes, use the ppcxx notation.